### PR TITLE
Migration models

### PR DESCRIPTION
**Pull request by at-hungngo**
Create migration:
1. Bids
2. Roles
3. Orders

Create models:
1. Bid
2. Role
3. Order

Edit migration and model:
1. User

### CODE REVIEW

File: database/migrations/2017_04_17_045658_create_orders_table.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        Schema::create('orders', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->text('description');
+            $table->float('weight')->default(0);
+            $table->float('price_bid')->default(0);
+            $table->string('reciver_name');
+            $table->string('reciver_address');
+            $table->string('reciver_phone');
+            $table->string('lnglat_start');
```
Comment (dzungnguyenh) [inline database/migrations/2017_04_17_045658_create_orders_table.php]: [rank B] I think these coordinates should be separate as independency field: latitude and longitude

File: database/migrations/2017_04_17_045628_create_bids_table.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('bids', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('description');
+            $table->float('price_bid')->default(0);
+            $table->integer('user_id')->unsigned();
```
Comment (dzungnguyenh) [inline database/migrations/2017_04_17_045628_create_bids_table.php]: [rank C] please create foreign relationship in migration, if any.

File: database/migrations/2017_04_17_045658_create_orders_table.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            $table->string('lnglat_end');
+            $table->tinyInteger('status');
+            $table->integer('user_id')->unsigned();
+            $table->integer('shipper_id')->unsigned();
```
Comment (dzungnguyenh) [inline database/migrations/2017_04_17_045658_create_orders_table.php]: [rank C] please create foreign relationship in migration, if any.

**Review by dzungnguyenh — CHANGES_REQUESTED** [review]
please check my comments.

File: database/migrations/2017_04_17_045658_create_orders_table.php (hunk 1 above)
Comment (at-hungngo) [reply database/migrations/2017_04_17_045658_create_orders_table.php]: I got it.

**Review by dzungnguyenh — CHANGES_REQUESTED** [review]
please use soft delete for all models.

**Review by dzungnguyenh — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**